scripts/run.sh

```diff
@@ -1,23 +1,3 @@
 #!/bin/bash
 
-# Start the first process & redirect output to a temporary file
-./flow-x86_64-linux- emulator --evm-enabled > temp_output.txt &
-
-# PID of the first process
-FIRST_PROCESS_PID=$!
-
-# Monitor the temporary file for a specific output
-PATTERN="3569"
-while ! grep -q "$PATTERN" temp_output.txt; do
-  sleep 1
-done
-
-# Once the pattern is found, you can kill the first process if needed
-# kill $FIRST_PROCESS_PID
-
-# Run the second process
-
-./flow-x86_64-linux- transactions send /flow-evm-gateway/create_bridged_account.cdc 1500.0 --network=emulator --signer=emulator-account && ./evm-gateway
-
-# Clean up temporary file
-rm temp_output.txt
+./flow-x86_64-linux- transactions send /flow-evm-gateway/create_bridged_account.cdc 1500.0 --network=$FLOW_NETWORK --signer=emulator-account && ./evm-gateway
```

Comment:
The `--network=$FLOW_NETWORK` has to be passed to the `./evm-gateway`.
The transaction part is only applicable in case where we want to setup the Emulator environment.
In the rest of the environments, the creation of a COA account will be a one-off operation.

Reply:
technically we won't need this, because the env vars will be pulled directly from the cloud run

Reply:
Exactly :+1:

Reply:
Can we hold off to deploying this though? So that we have a working version deployed, with which devs can interact with.

Reply:
I'm waiting until @sideninja has finished things on his side when we can revisit this to make sure everything lines up

Reply:
Sounds good :+1: